api.yaml

```diff
@@ -304,30 +114,92 @@ parameters:
     type: string
     description: ID of the application
     required: true
-    pattern: "^[a-z][a-z0-9-]*[a-z0-9]$"
-    #example: kio
 
-  VersionID:
-    name: version_id
-    in: path
+  StartDate:
+    name: from
+    in: query
+    description: date specifying to only return events that occured past (inclusive) that date.
+    required: false
```

Comment:
what is the default, when a user did not specify any date? maybe this param, and EndDate as well, should be required